community/csv/CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+2.2.2
+-----
+o BufferedCharSeeker has ability to allow/disallow multi-line fields
```

Comment:
I wrote about that in import-tool/CHANGES.txt because that's the more user-facing part of this whole thing. There the default is stated.